lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -64,6 +70,21 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
         Please create a new one by clicking <span class="font-semibold">“New notebook”</span>
       </div>
     </div>
+    <%= if @show_autosave_note? do %>
+      <div class="mt-4 p-5 flex space-x-4 items-center border border-gray-200 rounded-lg">
+        <div>
+          <.remix_icon icon="history-line" class="text-gray-400 text-xl" />
+        </div>
+        <div class="text-gray-600">
+          We automatically persist all notebooks that you forget to save manually.
+          <br>
+          To restore one of those
+          <a class="font-semibold" href="#" phx-click="open_autosave_pathectory">
+            go to backups
+          </a>
```

Comment:
```suggestion
          </a>.
```
?

Reply:
That was for consistency, or do we want the dot for both?

![image](https://user-images.githubusercontent.com/17034772/144712519-d890fe6d-236f-4787-8fc7-aa48729b42c2.png)

Reply:
Ah, I would have the dot on both but I am wondering if we should have a single message. There is another inconsistency that "new notebook"  is in bold but not a button. Go to backups is in bold, but a button. Perhaps we want both to be consistent and perhaps use an actual color?

In any case, what do you think about:

> You do not have any running sessions.
> Create a [new notebook].
> Looking for unsaved notebooks? [Browse them here].

Reply:
Another option is:

```
You do not have any running sessions.                   |--------------|
                                                        | New notebook |
Looking for unsaved notebooks? [Browse them here].      |--------------|
```

Reply:
![image](https://user-images.githubusercontent.com/17034772/144713331-a9abf56a-285e-42c9-8742-547c51d4c3b1.png)

![image](https://user-images.githubusercontent.com/17034772/144713427-585121b4-1a69-4056-85d3-beac799c8a99.png)

The first one is definitely too dense, but the second one looks pretty neat. Not sure if we should have two ways of creating a new notebook, but it's probably fine?

WDYT?

Reply:
Love the second one, :shipit: !

Reply:
Oh, and what happens when we don't show "Browse them here."? Is the text still centralized?

Reply:
![image](https://user-images.githubusercontent.com/17034772/144714979-98cf63eb-9d5e-48d4-a866-006faea1df45.png)